recipes/Flask-Minify/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "flask-minify" %}
+{% set version = "0.42" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Flask-Minify-{{ version }}.tar.gz
+  sha256: cdae9bd492b4200ebbf9fd0b08878452274ddca454f3eb1dff8b900629538c8b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.0
+  run:
+    - flask
+    - htmlmin
+    - jsmin
+    - lesscpy
+    - python >=3.0
```

As per https://github.com/mrf345/flask_minify/blob/master/setup.py#L5

```suggestion
    - python >=3.6
```